_includes/nav.html

```diff
@@ -1,3 +1,3 @@
 <div>
-    <a class="internal-link" href="/"><b>{{ site.title }}</b></a>
+    <a class="internal-link" href="{{site.baseurl}}/"><b>{{ site.title }}</b></a>
```

Comment:
For formatting consistency, we could insert spaces around `site.baseurl`:

```suggestion
    <a class="internal-link" href="{{ site.baseurl }}/"><b>{{ site.title }}</b></a>
```